### DISCUSSION TITLE
Data set query on the use of LMFlow for psychological accompaniment tasks

### DISCUSSION THREAD

**Discussion by zcm200605** [OptimalScale/LMFlow#508] [Q&A] [answered]
![05ecbc45d7a32e64ca71b38829f02f7](https://github.com/OptimalScale/LMFlow/assets/65277046/c971792c-05d2-407d-80f1-34a123f5bfe5)
I would like to ask you, I saw an example of using lmflow for psychological accompaniment in an open course and at that time the teacher said that the dataset was available on the official website. I would like to ask which dataset is it, I can't seem to find it?

**Comment by shizhediao** [comment, marked as answer]
Hi
The dataset for psychological accompaniment is not released due to some concerns.
If you want to train a general chatbot model, you may use this one: [lmflow_data.tar.gz](http://lmflow.org:5000/lmflow_data.tar.gz)
